src/NSwag.Sample.NET5/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+
+namespace NSwag.Sample.NETCore31.Controllers
```

Comment:
For consistency namespace should be named probably NET50.

Reply:
Done